requirements.txt

```diff
@@ -0,0 +1,14 @@
+beautifulsoup4==4.11.1
+huggingface-hub==0.11.1
+requests==2.28.1
+tensorboard==2.11.0
+tensorboard-data-server==0.6.1
+tensorboard-plugin-wit==1.8.1
+tensorflow-estimator==2.11.0
+tensorflow-io-gcs-filesystem==0.28.0
```

Comment:
I guess you never use these packages. Moreover, HF uses PyTorch by default, so there is no need for TensorFlow at all.

Reply:
It was my mistake, removed them